main.js

```diff
@@ -294,6 +321,25 @@ export class IdDecoder {
         bitLength: this.fixedBitLength
       });
     }
+    if(this.multihash) {
+      // <varint hash fn code>: identity function
+      const [hashFnCode] = decoded.slice(0, 1);
```

Comment:
Isn't this simpler? Or am I off?

```suggestion
      const [hashFnCode] = decoded;
```

Reply:
thanks, addressed 30d05b7